apps/gateway/.env.example

```
@@ -0,0 +1,19 @@
+# Gateway Environment Variables
+
+# ── Crypto ──────────────────────────────────────────────────────────────────
+# 64 hex chars = 32 bytes AES-256-GCM key
+GATEWAY_ENCRYPTION_KEY=
+
+# Internal API auth token
+INTERNAL_API_TOKEN=
+
+# ── Telegram Adapter (F3a-18) ────────────────────────────────────────────────
+TELEGRAM_MODE=webhook          # webhook | polling (default: webhook)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)